buildtest/utils/file.py

```diff
@@ -57,6 +57,28 @@ def is_dir(dirname):
     return os.path.isdir(dirname)
 
 
+def is_symlink(sym_link):
+    """Check if the given link is a symlink and return ``True``, if it is not a symlink or the symlink is broken return ``False``.
+
+    Args:
+        sym_link (str): input link to check.
+
+    Returns:
+        bool: True if input link is a symlink and is not broken otherwise return False.
```

Comment:
change to ` bool: True if filename is a symbolic link otherwise return False if its not a symbolic link or broken link`

Remember to use the full word `symbolic link` when referring to documentation (docstring) as pose to `symlink`